lib/routes/1lou/index.ts

```diff
@@ -18,7 +18,11 @@ export const handler = async (ctx) => {
 
     const currentUrl = new URL(`${params && params.endsWith('.htm') ? params : `${params}.htm`}${queryString ? `?${queryString}` : ''}`, rootUrl).href;
 
-    const { data: response } = await got(currentUrl);
+    const { data: response } = await got(currentUrl, {
+        headers: {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36',
```

Comment:
RSSHub will use a randomised UA of [Chrome on Mac](https://github.com/DIYgod/RSSHub/blob/02963d1951c4a076cfc34af05083c1ee2016dbdb/lib/config.ts#L453). Does the site only work with this fixed version of Chrome on Windows?